extension/src/popup/views/__tests__/SignTransaction.test.tsx

```diff
@@ -218,12 +218,15 @@ describe("SignTransactions", () => {
 
     expect(
       opDetails.includes(
-        `Parameters${args?.from.toString()}${args?.to.toString()}${args?.amount.toString()}`,
```

Comment:
These strings look like this now because of the tooltip added by the CopyContext component, it renders it inline with your element and show/hides.